src/style.css

```
@@ -0,0 +1,64 @@
+.form-field {
+  width: 100%;
+}
+.bigSearch {
+  height: 70px;
+  width: 900px;
+  left: 0px;
+  top: 0px;
+  border-radius: 4px;
+  padding-left: 50px;
+  font-weight: 300;
+  display: block;
+  margin-top: 20px;
+  border: 2px solid lightgray;
+}
+.pic1 {
+  height: 19px;
+  width: 19px;
+  left: 26px;
+  top: 48px;
+  border-radius: 0px;
+  box-sizing: border-box;
+  position: absolute;
+}
+.smallSearch {
+  display: block;
+  height: 50px;
+  width: 900px;
+  left: 0px;
+  top: 0px;
+  border-radius: 4px;
+  padding-left: 50px;
+  font-weight: 300;
+  margin-top: 20px;
+  border: 2px solid lightgray;
+  font-size: small;
+}
+.pic2 {
+  height: 15px;
+  width: 15px;
+  left: 26px;
+  top: 135px;
+  border-radius: 0px;
+  box-sizing: border-box;
+  position: absolute;
+}
+input:hover {
+  background-color: lightpink;
+}
+input:focus {
+  color: darkslateblue;
+}
+::placeholder {
+  font-family: Avenir;
+}
```

Add class to inputs and use class instead of the tag.
Also, You should also write the element to which you are applying the pseudo-element.
[HINT](https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder)
```suggestion
}
input:hover {
  background-color: lightpink;
}
input:focus {
  color: darkslateblue;
}
.smallSearch::placeholder {
  font-family: Avenir;
}
```